### DISCUSSION TITLE
Drafts and Notifications

### DISCUSSION THREAD

**Discussion by JonathanAsbury-ACS** [community/community#3462] [Discussions]
Communication across our discussions is important to keep everyone informed  so our community members subscribe to notifications on Discussions.  Unfortunately doing so gets noisy and brings attention to content that isn't often relevant or ready for feedback.  It would be ideal if there was control over when notifications go out and content can be prepared without blasting users about work in progress.  

Perhaps allowing a draft/publish version would be an intuitive option.  Once published the notification would be sent but never while in draft.  

Keep up the great work.  Powerful tool.

**Comment by PranamBhat** [comment]
Great input. @JonathanAsbury-ACS  Thank you!
